knowledge-base/karancode/yamllint-github-action/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Yamllint Github Action'
+github-token:
+  environment-variable-name: GITHUB_ACCESS_TOKEN
+  permissions:
+   issues: write
+   issues-reason: to post comments on PRs #Checkout: https://github.com/karancode/yamllint-github-action#secrets
```

Comment:
@h0x0er since it is posting to PR comment, should it not be `pull-requests: write`?